src/hallway/routes.clj

```diff
@@ -36,9 +36,8 @@
                        (uri/query-encode (str url " @FindkaEssays")))}
       "Twitter"]]
     [:div
-     [:a.link {:href (str "http://localhost:8080/rss?url="
-                       (uri/query-encode url))}
-      "Discussion RSS Feed"]]))
+     [:a.link {:href (str "/feed?url=" (uri/query-encode url))}
+      "Atom feed"]]))
```

Comment:
Using `localhost` won't work in production. We could use `base-url` here as in the feed, but in this case a relative url works fine too.